sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/optimizer/RewriteDistinctAggregates.scala

```diff
@@ -331,6 +457,17 @@ object RewriteDistinctAggregates extends Rule[LogicalPlan] {
     }
   }
 
+  /**
+   * Collect all aggregate expressions.
```

Comment:
We don't need this comment (it is totally the same with the function name below...

Reply:
OK